### DISCUSSION TITLE
Debug JSON error on remote schema

### DISCUSSION THREAD

**Discussion by edouardouvrard** [hasura/graphql-engine#6741] [Question]
Hi, 
I'm getting this error from a remote schema query : 
`Received invalid JSON value from remote`

How can i debug the raw response received by hasura that causing this error ? 
Log on my server show a valid json.

Thanks

**Comment by mingfang** [comment]
I'm also seeing this kind of problem when querying a remote Weaviate API https://github.com/semi-technologies/weaviate.
```json
{
  "type":"http-log",
  "timestamp":"2021-04-11T22:38:12.583+0000",
  "level":"error",
  "detail":{
    "operation":{
      "user_vars":{"x-hasura-role":"admin"},
      "error":{
        "path":"$",
        "error":"Received invalid JSON value from remote",
        "code":"remote-schema-error"
      },
      "request_id":"0bd43c27d103a1069c91e66c5d5c780e",
      "response_size":91,
      "query":{
         "operationName":"MyQuery",
         "query":"query MyQuery {\n  Aggregate {\n    Article {\n      title {\n        count\n      }\n    }\n  }\n}\n"}},
         "request_id":"0bd43c27d103a1069c91e66c5d5c780e",
         "http_info":{
            "status":200,
            "http_version":"HTTP/1.1",
            "url":"/v1/graphql",
            "ip":"192.168.2.162","method":"POST","content_encoding":null
          }
      }
}  
```
The same query works when hitting Weaviate directly.

**Reply by edouardouvrard** [reply]
Hi @mingfang 
I think i've found a solution for me.
My json response have an `extensions` object that seems to be ok for the graphql spec but not for hasura, when i remove this object from the response all it's working fine.

**Reply by mingfang** [reply]
@edouardouvrard I also found the problem with my json.
But I think your original point is still valid, Hasura needs dump the response in the event of a json parsing error
